ScoutSuite/providers/azure/resources/sql/servers.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+from azure.mgmt.sql import SqlManagementClient
+
+from ..resources import AzureCompositeResources
+from ScoutSuite.providers.azure.utils import get_resource_group_name
+
+from .databases import Databases
+from .server_azure_ad_administrators import ServerAzureAdAdministrators
+from ..utils import get_non_provider_id
```

Comment:
I believe we've been mostly using absolute paths. I find it's more verbose, especially for very vague names like utils.

Reply:
I agree for `..utils`. Although, I would suggest to use relative paths to refer to files of a same directory and use absolute paths for all other cases (when you need to import a file that is at least one level above in the file tree). That way, we're still taking advantages of Python package system to write imports like `from .database_blob_auditing_policies import DatabaseBlobAuditingPolicies` instead of `from ScoutSuite.providers.azure.resources.sqldatabase.database_blob_auditing_policies import DatabaseBlobAuditingPolicies`.